src/tweet.js

```diff
@@ -237,21 +237,28 @@ export default class Tweet {
   }
 
   _updateLinks(div, tweetData) {
-    let textWithoutLinks = tweetData.text || tweetData.full_text;
+    let textWithLinks = tweetData.text || tweetData.full_text;
 
     for (const urlIndex in tweetData.entities.urls) {
       const url = tweetData.entities.urls[urlIndex].url;
-      textWithoutLinks = textWithoutLinks.replace(url, '');
+      textWithLinks = textWithLinks.replace(url, this._getLinkSpan(url).outerHTML);
+    }
+    if (tweetData.extended_entities) {
+      for (const mediaIndex in tweetData.extended_entities.media) {
+        const url = tweetData.extended_entities.media[mediaIndex].url;
+        textWithLinks = textWithLinks.replace(url, this._getLinkSpan(url).outerHTML);
```

Comment:
@rodrigopavezi is there a use case to display the media as a link? I don't think it aggregates much to the people seeing the twitter on displays.
Besides as per the example on the [twitter documentation](https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/extended-entities-object), all media will point to the same url, which is the tweet url.

Reply:
@rudibravo Are you suggesting just hiding the URL?

Reply:
@sheadarlison Yes, I don't see much value on showing some pictures URLs.

Reply:
@rudibravo yeah I agree with that. @rodrigopavezi could you hide image urls but make sure links to other websites still show with the blue color.

Reply:
@sheadarlison @rudibravo sure, that makes sense. cheers